go.mod

```diff
@@ -66,8 +66,6 @@ require (
 	github.com/pelletier/go-toml v1.6.0 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.4.0
-	github.com/prometheus/client_model v0.2.0
```

Comment:
Hm, when I run `go mod tidy` locally, this is includes. Any ideas why?

```
$ go version
go version go1.13.7 linux/amd64
```

Reply:
That's what the bug is about which I filed. If you run go mod tidy, it changes go.mod. But if you run go mod vendor again, it reverts those changes. They are in conflict with each other. I just went with one version.

Reply:
Got it. Thanks for point out that this is caused by https://github.com/golang/go/issues/37376